classes/class-p4-search.php

```diff
@@ -384,7 +388,7 @@ protected function get_posts( $paged = 1 ) : array {
 
 			// This happens when we search for everything or when filtering for attachments, since WP_Query does not support searching within rich text documents.
 			if ( ! $this->search_query || ( isset( $args['post_type'] ) && 'attachment' !== $args['post_type'] ) ) {
-				if ( $posts ) {
+				if ( isset($posts) ) {
```

Comment:
This change is unrelated to the issue, but the line was causing: `Notice: Undefined variable: posts in /app/source/public/wp-content/themes/planet4-master-theme/classes/class-p4-search.php on line 387`.